miden/tests/integration/operations/field_ops.rs

```diff
@@ -683,6 +719,22 @@ proptest! {
         build_op_test!(asm_op, &[b as u64]).prop_expect_stack(&[expected])?;
     }
 
+    #[test]
+    fn ilog2_proptest(b in 1..u64::MAX) {
+        let asm_op = "ilog2";
+        let ilog2 = b.ilog2();
+        let pow2 = 2u64.pow(ilog2);
+
+        build_op_test!(asm_op, &[b]).prop_expect_stack(&[pow2, ilog2.into()])?;
+    }
+
+    #[test]
+    fn trailing_ones_proptest(b in 1..u64::MAX) {
```

Comment:
I don't see a clear reason why skip `0` here as it should be a valid input for `trailing_ones`

Reply:
copy-pasta mistake, fixed